tests/sentry/api/endpoints/test_organization_dashboards.py

```diff
@@ -1879,3 +1880,16 @@ def test_order_by_most_favorited(self) -> None:
             "Dashboard 3",
             "Dashboard 1",
         ]
+
+    @patch("sentry.quotas.backend.get_dashboard_limit")
+    def test_dashboard_limit_prevents_creation(self, mock_get_dashboard_limit) -> None:
+        mock_get_dashboard_limit.return_value = 1
+        with self.feature("organizations:dashboards-plan-limits"):
+            response = self.do_request("post", self.url, data={"title": "New Dashboard w/ Limit"})
+        assert response.status_code == 400
+        assert response.data == "You may not exceed 1 dashboards on your current plan."
```

Comment:
"1 dashboards" is okay here because in production, we're going to apply limits of `10` and `20`